design/fast_finality.md

```diff
@@ -0,0 +1,262 @@
+# Fast finality design
+
+## Scope
+
+The introduction of a single sequencer to improve cost-per-transaction during the bootstrapping phase, as 
+described in the [Bootstrapping Strategy design doc](./Bootstrapping_strategy.md).
+
+## Glossary
+
+* *Light batch*: A set of transactions considered to be soft-finalised
+
+## Requirements
+
+* Finality
+  * Transaction *soft* finality (finality guaranteed by the sequencer) is achieved in under one second
+  * There is eventual transaction *hard* finality (finality guaranteed by the L1)
+  * The sequencer achieves hard finality on an agreed cadence
+  * Transactions are hard-finalised in the same order they are soft-finalised
```

Comment:
create an incentive based mechanism to ensure that ...

Reply:
Fixed